src/modules/commander/Arming/PreFlightCheck/PreFlightCheck.cpp

```diff
@@ -45,15 +45,11 @@
 bool PreFlightCheck::preArmCheck(orb_advert_t *mavlink_log_pub, const vehicle_status_flags_s &status_flags,
 				 const safety_s &safety, const uint8_t arm_requirements)
 {
-	bool reportFailures = true;
 	bool prearm_ok = true;
 
-
 	// USB not connected
 	if (!status_flags.circuit_breaker_engaged_usb_check && status_flags.usb_connected) {
-		if (reportFailures) {
-			mavlink_log_critical(mavlink_log_pub, "Arming denied! Flying with USB is not safe");
-		}
+		mavlink_log_critical(mavlink_log_pub, "Arming denied! Flying with USB is not safe");
```

Comment:
I might be missing some of the high level design, but isn't this potentially a step in the wrong direction?

Don't we ultimately want to be able to know pre-arm check status continuously, and separately get the relevant messages as needed?

Reply:
Yes, it's just the start that looks into the wrong direction becuase the reportFailures flag was a local variable that was never changed and I plan to only have the output once you know what checks have failed and not spread over all the checks.

I can revert this change since it's anyways not how it ends up to be and limit the focus of this pr on the raw structure.

Reply:
I removed the functional changes since this should be pure refactoring and the functional changes should be a real step in the right direction before merging.